src/cli/cli_network_data.cpp

```diff
@@ -655,7 +676,40 @@ otError NetworkData::OutputBinary(bool aLocal)
  * @par
  * `netdata show` from OT CLI gets full Network Data received from the Leader. This command uses several
  * API functions to combine prefixes, routes, and services, including #otNetDataGetNextOnMeshPrefix,
- * #otNetDataGetNextRoute, and #otNetDataGetNextService.
+ * #otNetDataGetNextRoute, #otNetDataGetNextService and #otNetDataGetNextLowpanContextInfo.
+ * @par
+ * On-mesh prefixes are listed under `Prefixes` header:
+ * * The on-mesh prefix
+ * * Flags
+ *   * p: Preferred flag
+ *   * a: Stateless IPv6 Address Autoconfiguration flag
+ *   * d: DHCPv6 IPv6 Address Configuration flag
+ *   * c: DHCPv6 Other Configuration flag
+ *   * r: Default Route flag
+ *   * o: On Mesh flag
+ *   * s: Stable flag
+ *   * n: Nd Dns flag
+ *   * D: Domain Prefix flag (only available for Thread 1.2).
+ * * Preference `high`, `med`, or `low`
+ * * RLOC16 of device which added the on-mesh prefix
+ * External Routes are listed under `Routes` header:
```

Comment:
Can you add a `@par` line before this one

Otherwise it includes it in the above list in the import

Reply:
Changed in new push.